app/web/src/api/sdf/dal/schema.ts

```diff
@@ -66,6 +66,8 @@ export interface SchemaVariant {
   outputSockets: OutputSocket[];
   props: Prop[];
   canCreateNewComponents: boolean;
+
+  canUpdate: boolean;
```

Comment:
This has been moved to the variant itself and is no longer mutated in `app/web`.